src/js/tracing/reactnativenavigation.ts

```diff
@@ -20,10 +20,18 @@ interface ReactNativeNavigationOptions {
    * Default: 1000
    */
   routeChangeTimeoutMs: number;
+  /**
+   * Instrumentation will create a transaction on tab change.
+   * By default only navigation commands create transactions.
+   *
+   * Default: false
+   */
+  enableTabsInstrumentation: boolean;
 }
 
 const defaultOptions: ReactNativeNavigationOptions = {
   routeChangeTimeoutMs: 1000,
+  enableTabsInstrumentation: false,
```

Comment:
I thought it could be an issue for people who are using the integration now, as it might generate more transactions. 
But when I'm thinking about it now, those transactions will be more correct, at if they don't like it they can turn it off.

Reply:
yep, I'd consider as a new feature but telling how to turn it off.